pypeit/par/pypeitpar.py

```diff
@@ -1602,11 +1602,21 @@ def __init__(self, obj_toler=None, assign_obj=None,
         dtypes['assign_obj'] = bool
         descr['assign_obj'] = 'If SlitMask object was generated, assign RA,DEC,name to detected objects'
 
-        defaults['slitmask_offset'] = 0.
+        defaults['nsig_thrshd'] = 50.
+        dtypes['nsig_thrshd'] = [int, float]
+        descr['nsig_thrshd'] = 'Objects detected above this significance threshold will ' \
+                               'be use to compute the slitmask offset. This is ignored if ' \
+                               '``slitmask_offset`` or ``bright_maskdef_id`` is set.'
+
+        defaults['slitmask_offset'] = None
```

Comment:
I don't understand why the user has to provide these parameters? Isn't the point to determine these numbers automatically? Perhaps it is just a question of the docs, i.e. are these parameters going to usually be None, and then if they are set they are used overriding the automated estimation?

Reply:
Just clarify that these are optional and it will do something automated that is sensible if they are not provided.